app/services/page_repository.rb

```diff
@@ -0,0 +1,33 @@
+class PageRepository
+  class << self
+    def find(page_id:, form_id:)
+      Page.find(page_id, params: { form_id: })
+    end
+
+    def create!(record)
+      page = Page.new(record.attributes, false)
+      page.prefix_options = record.prefix_options
+      page.save # rubocop:disable Rails/SaveBang
```

Comment:
I think we can change this to `save!`

```suggestion
      page.save!
```

Reply:
Sorted